modules/bizzclickBidAdapter.js

```diff
@@ -0,0 +1,317 @@
+const NATIVE_ASSET_IDS = { 0: 'title', 2: 'icon', 3: 'image', 5: 'sponsoredBy', 4: 'body', 1: 'cta' };
+const NATIVE_PARAMS = {
+  title: {
+    id: 0,
+    name: 'title'
+  },
+  icon: {
+    id: 2,
+    type: 1,
+    name: 'img'
+  },
+  image: {
+    id: 3,
+    type: 3,
+    name: 'img'
+  },
+  sponsoredBy: {
+    id: 5,
+    name: 'data',
+    type: 1
+  },
+  body: {
+    id: 4,
+    name: 'data',
+    type: 2
+  },
+  cta: {
+    id: 1,
+    type: 12,
+    name: 'data'
+  }
+};
+const NATIVE_VERSION = '1.2';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: (bid) => {
+    return Boolean(bid.params.accountId) && Boolean(bid.params.placementId)
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: (validBidRequests, bidderRequest) => {
+    if (validBidRequests.length === 0) return []
+    let accuontId = validBidRequests[0].params.accountId;
+    const endpointURL = URL_ENDPOINT.replace(ACCOUNTID_MACROS, accuontId);
+
+    let winTop = window;
+    let location;
+    try {
+      location = new URL(bidderRequest.refererInfo.referer)
+      winTop = window.top;
+    } catch (e) {
+      location = winTop.location;
+      utils.logMessage(e);
+    };
+
+    let bids = [];
+    for (let bidRequest of validBidRequests) {
+      let impObject = prepareImpObject(bidRequest);
+      let data = {
+        id: bidRequest.bidId,
+        test: config.getConfig('debug') ? 1 : 0,
+        cur: ['USD'],
+        device: {
+          w: winTop.screen.width,
+          h: winTop.screen.height,
+          language: (navigator && navigator.language) ? navigator.language.indexOf('-') != -1 ? navigator.language.split('-')[0] : navigator.language : '',
+        },
+        site: {
+          page: location.pathname,
+          host: location.host
+        },
+        source: {
+          tid: bidRequest.transactionId
+        },
+        tmax: bidRequest.timeout,
+        imp: [impObject],
+      };
+      bids.push(data)
+    }
+    return {
+      method: 'POST',
+      url: endpointURL,
+      data: bids
+    };
+  },
+
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: (serverResponse) => {
+    try {
+      serverResponse = serverResponse.body;
```

Comment:
Consider: 

```
if (!serverResponse || !serverResponse.body) return []
serverResponse = serverResponse.body
```

Anyway, it's generally considered a bad practice to modify the input parameter of a function. You may think of assigning it to a newly created variable.